src/components/PathwayExercise/index.js

```diff
@@ -114,12 +116,17 @@ function PathwayExercise() {
     })
       .then((res) => {
         setCourse(res.data.course.exercises);
+        setAvailableLang(res.data.course.lang_available);
       })
       .catch((err) => {
         console.log("error");
       });
   }, []);
-
+  const Lang = {
+    en: "English",
+    mr: "Marathi",
+    hi: "Hindi",
+  };
```

Comment:
Can you import the language map constant from here: https://github.com/navgurukul/bhanwari-devi/blob/08eaeab3030736341d3b7a3685c9477e3bb40de6/src/pages/CourseContent/languageMap.js#L1-L7 instead even if it currently contains a couple of languages (Tamil and Telugu) for which there are no translations?